src/components/Button/Button.js

```diff
@@ -1,15 +1,17 @@
 import styled from 'styled-components/macro'
 
 export default styled.button`
-  border: 1px solid orange;
-  width: 30%;
-  padding: 8px;
+  border: 0.5px solid grey;
```

Comment:
Why don't you use your color variable here?
```suggestion
  border: 0.5px solid var(--color-grey);
```